springwolf-plugins/springwolf-sqs-plugin-binding/src/main/java/io/github/springwolf/plugins/sqs/configuration/SpringwolfSqsBindingAutoConfiguration.java

```diff
@@ -0,0 +1,34 @@
+// SPDX-License-Identifier: Apache-2.0
+package io.github.springwolf.plugins.sqs.configuration;
```

Comment:
For the `1.0.0` release, each gradle module has its own package namespace.

Idea:
```suggestion
package io.github.springwolf.bindings.sqs.configuration;
```

Similarly, I wonder whether the gradle module should be outside of the `springwolf-plugins` folder and have its own `springwolf-bindings` folder.

I am happy to discuss and also get feedback from @sam0r040

Reply:
@timonback I don't think we can use `@AliasFor` since it doesn't support `@interface`

I will go with a full duplication and add a @deprecation to the existing annotations (timing is kind of bad)

I create its own pipeline for the `bindings` because there is no guarantee parity, but then it may make more sense to remove the `plugin` reference to the artifact.

Reply:
Not sure where we continue our discussion, but I'll comment here. I think structure wise it looks good now, but the duplication worries me a bit. Did you think about changing the annotations in the plugins to meta annotations which would just "redirect" to the new ones? If our resolution mechanism can handle them, thats what I would vote for.

Reply:
@sam0r040 I have tried using the meta annotations approach but I'm facing some limitations that I don't see how to address (check the error with the sns-example).

Reply:
Yes @ctasada , neither do I see a way to resolve the annotation in annotation issue.
Didn't expect that to be honest...

I guess the `AliasFor` approach won't be possible then and we have to go with breaking (import) changes -> removing the annotations for the `plugin` and keeping them in the `binding` only.

In terms of structure, the PR looks good to me. I am positive that it can be part of the next release.